### ISSUE TITLE
Vermicomposting Toilet (Water Flushing) 🚽 💧 🤞 

### ISSUE THREAD

**Issue by nelsonic** [dwyl/phase-three#150]
If we are able to buy land #116 the _first_ thing we will need to build is a _toilet_
so I've been extending my research into [**septic tanks**](https://en.wikipedia.org/wiki/Septic_tank), [sawdust] [composting toilets](https://en.wikipedia.org/wiki/Composting_toilet) 
and **vermicomposting _flush_ toilets**. 

http://www.vermicompostingtoilets.net
![image](https://github.com/user-attachments/assets/0d0ce452-f2fb-4de6-b33e-9867096ebb0e)

The plans are _very_ detailed: 
http://www.vermicompostingtoilets.net/design-construction/
[Design-and-construction-of-a-vermicomposting-toilet-_-Vermicomposting-Toilets.pdf](https://github.com/user-attachments/files/16738274/Design-and-construction-of-a-vermicomposting-toilet-_-Vermicomposting-Toilets.pdf)

Crucially, these are approved for use in off-grid applications in `Portugal` with _plenty_ of precedent:
http://www.vermicompostingtoilets.net/projects/
![image](https://github.com/user-attachments/assets/aaf7b333-3e0c-490c-9c09-6dd73d63769c)

**Comment by nelsonic** [comment]
@carolrmc please have a look at this and add anything you find or ask any follow-up questions you have.

**Comment by carolrmc** [comment]
If possible a visit would be a good idea

**Comment by nelsonic** [comment]
Indeed, it would be awesome to organise a visit to one of these farms in PT that has the composting toilet setup. 💡
We need to draft an email/message to reach out to them explaining our plans and ask if we can visit.
